Safewalk initial pack (#12793)
demisto/content: Packs/Safewalk/Integrations/SafewalkManagement/SafewalkManagement_description.md

## Create an API token.

> The following steps generates an API token key to be able to comunicate with your Safewalk server

Reviewer: ```suggestion
An API token is required for communicating with your Safewalk server.
```

To create an API token, follow these steps:

> - Login in to your Safewalk server usign a SSH client.

Reviewer: ```suggestion
- Log in to your Safewalk server using an SSH client.
```
